zerver/actions/create_user.py

```diff
@@ -551,6 +552,12 @@ def do_reactivate_user(user_profile: UserProfile, *, acting_user: Optional[UserP
                 }
             ).decode(),
         )
+
+        if user_profile.full_name.endswith(" (spammer)"):
+            check_change_full_name(
+                user_profile, user_profile.full_name.replace(" (spammer)", ""), acting_user
+            )
```

Comment:
@alya is removing "(spammer)" from the name of a user when reactivating them part of the design for this feature?

I would prefer we not do so, both because that's not going to work with i18n if we translate "spammer", which we probably should, and also because that feels like a separate operation.